cassandra-analytics-integration-framework/src/main/java/org/apache/cassandra/testing/TestUtils.java

```diff
@@ -36,7 +36,7 @@ public final class TestUtils
     private static final AtomicInteger TEST_TABLE_ID = new AtomicInteger(0);
 
     //    public static final int ROW_COUNT = 10_000;
-    public static final int ROW_COUNT = 1_000;
+    public static final int ROW_COUNT = 100;
```

Comment:
can you revert the change? I assume you lower the row count for local testing.

Reply:
I ran the tests after reverting `ROW_COUNT`, cannot reproduce timing out. Maybe it was related to the deep copy for all columns (which is now resolved). Can you revert the change?

Reply:
this change should be reverted. Are you seeing issues after the new change?

Reply:
I am seeing quite slower runs, sometimes timeouts only when running under Intellij. Reverted this change.